### PR TITLE
Database stuff

### PR DESCRIPTION
**Pull request by itohatweb**
DON'T just merge this PR cause this will break the DB.

### CODE REVIEW

File: src/database/database.ts [hunk 1]
```diff
@@ -47,9 +53,16 @@ import {
 
 // Create the database class
 const sabr = new Sabr();
+sabr.directoryPath = `${
```
Comment (Skillz4Killz) [inline src/database/database.ts:56]: would this work if the directory structure is as follow

root
  skillz
    gamer-deno
    db

**Review by Skillz4Killz — COMMENTED** [review]
this might need to also be applied inside the services folder if im not mistaken

File: src/database/database.ts (hunk 1 above)
Comment (itohatweb) [reply src/database/database.ts:56]: Yes, if you follow a few rules at starting this works.

**Comment by itohatweb** [comment]
> this might need to also be applied inside the services folder if im not mistaken

No a simple import from `database.ts` is enough.

**Comment by itohatweb** [comment]
Maybe it is better to not merge this and instead change these files manually.

**Comment by Skillz4Killz** [comment]
I think using database file is not optimal here as everytime we start a service the entire script of fetching getAll of several tables need to run and then it also ends up creating several other deps and eating more memory than necessary

**Comment by itohatweb** [comment]
In this case we are not using `import()` but `import {db} from "database"`

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**